app/services/attribute_asserter.rb

```diff
@@ -15,10 +15,11 @@ class AttributeAsserter
     :phone
   ].freeze
 
-  def initialize(user, service_provider, authn_request)
+  def initialize(user, service_provider, authn_request, decrypted_pii)
     self.user = user
     self.service_provider = service_provider
     self.authn_request = authn_request
+    @decrypted_pii = decrypted_pii
```

Comment:
I noticed in many other places we prefix instance variables with `_`, so this would be `@_decrypted_pii` -- should we be doing this here?